.github/workflows/ci.yaml

```diff
@@ -0,0 +1,135 @@
+name: Lint and Test Charts
+
+on:
+  pull_request:
+    paths:
+      - 'charts/**'
+
+jobs:
+  lint-chart:
+    runs-on: ubuntu-latest
+    outputs:
+      list-changed: ${{ steps.list-changed.outputs.changed }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      # Required by helm/chart-testing-action
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      # Required by helm/chart-testing-action
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.1.0
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed)
+          echo "$changed"
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          else
+            echo "::set-output name=changed::false"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --lint-conf .github/ct.yaml
+
+  lint-docs:
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Run helm-docs
+        run: .github/helm-docs.sh
+
+  kubeval-chart:
+    runs-on: ubuntu-latest
+    needs:
+      - lint-chart
+    strategy:
+      matrix:
+        k8s:
```

Comment:
I guess this list will have to be maintained manually. Will this cause problems with linting failing in the future when using a future schema version (maybe that's not a problem and you should simply update this list)?

Reply:
These checks are here to ensure we keep backward compatability. So if I understand you correctly then those proposed issues are intentional. We want it to fail if we change to a future schema